.github/workflows/maestro_release.yaml

```diff
@@ -0,0 +1,27 @@
+name: Maestro Release
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+      - name: Install uv and activate the environment
+        uses: astral-sh/setup-uv@v6
+      - name: Build Maestro
+        shell: bash
+        run: uv build
+      - name: GitHub Release
+        uses: softprops/action-gh-release@v2
+        if: github.ref_type == 'tag'
+        with:
+          files: dist/*
+          generate_release_notes: true
```

Comment:
Would you add a sniff test step?  It is probably enough to install the package and run a existing test.

Reply:
done, I added a test job and made it a prerequisite for the release job, I also confirmed it works as expected on my fork

Reply:
I might not explain correctly what I want.  Can the package just build be tested?

Reply:
> Can the package just build be tested?

I was eventually able to add a test like in `maestro_test-workflow.yaml`, but our current pytests would require quite a bit of editing to support running against a distribution install rather than a dev install as they were intended.